### PR TITLE
[NOTEPAD] Add Croatian translation (hr-HR)

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose
Translate Notepad to Croatian
JIRA issue: N/A

## Proposed changes
- Upload hr-HR.rc

## TODO
Nothing :)

## Note
This is new Pull Request because [old one](https://github.com/reactos/reactos/pull/4330) had conflict.

### CODE REVIEW

File: base/applications/notepad/lang/hr-Hr.rc [hunk 1]
```diff
@@ -0,0 +1,171 @@
+ LANGUAGE LANG_CROATIAN, SUBLANG_DEFAULT
```
Comment (katahiromz) [inline base/applications/notepad/lang/hr-Hr.rc:1]:
```suggestion
LANGUAGE LANG_CROATIAN, SUBLANG_DEFAULT

```

File: base/applications/notepad/rsrc.rc [hunk 2]
```diff
@@ -143,6 +146,9 @@ IDI_NPICON ICON "res/notepad.ico"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_ZH_HK
```
Comment (katahiromz) [inline base/applications/notepad/rsrc.rc:149]: Hong Kong translation is not added yet.

Comment (katahiromz) [reply base/applications/notepad/rsrc.rc:149]: Never mind. My mistake.

File: base/applications/notepad/lang/hr-Hr.rc [hunk 3]
```diff
@@ -0,0 +1,172 @@
+LANGUAGE LANG_CROATIAN, SUBLANG_DEFAULT
```
Comment (binarymaster) [inline base/applications/notepad/lang/hr-Hr.rc:1]: Translation copyright header is missing, I'll add one.

**Review by binarymaster — COMMENTED** [review]
Resource file name has incorrect letter case, this has to be fixed. Sorry for late review, I wanted to do it earlier, but was on vacation.

image

**Comment by binarymaster** [comment]
Addressed in 015fd3a72f26987cb3aa33336991a466b30b2ca3.

**Pull request merged**